README.md

```diff
@@ -1 +1,32 @@
-# Assignment
```

Comment:
Remove this

Reply:
That is removed red lines stand for things that have been removed

Reply:
Sorry didn't notice the colour